### ISSUE TITLE
Status view for fetching ISO

### ISSUE THREAD

**Issue by shithead** [clonos/cp#19]
For create a bhyve vm it doesnt show porgess of fetch iso image. That make sometimes confuse in which state clonos is for the VM.

**Comment by olevole** [comment]
Yes, at the moment this problem exists. For this reason, this is mentioned here: 
https://clonos.tekroutine.com/errata.html

Some commands already have a status update via queue ( CBSD used ws:// to send events directly in ClonOS ). But many teams do not yet have this support. Work is slow, but continues.

My current plan: make queue subsystem in CBSD more unversal, e.g. add queue.conf so that users can receive events in any backen - into redis, or AMQP (rabbit,beanstalk, zmq) or ws:// and so on and continue to implement support for progressbar/updates in atomic in long operations ( fetch, import, export, clone ..)

**Comment by olevole** [comment]
By the way .. In CBSD there is a command fetch_iso ( https://www.bsdstore.ru/en/cbsd_fetch.html ) which can be used to pre-obtain images referenced by CBSD profiles,e.g:

(in latest CBSD, 12.0.8)

% cbsd fetch_iso dstdir=default keepname=0 cloud=0  ( for ISO )
% cbsd fetch_iso dstdir=default keepname=0 cloud=1  (for bhyve cloud images )
% cbsd fetch_iso dstdir=default keepname=0 cloud=1 conv2zvol=1  (for bhyve cloud images + auto convert them into ZVOL )

This can be used during the preparation of the hypervisor for work.
